src/view/terminalview.h

```diff
@@ -0,0 +1,17 @@
+
+#ifndef EVE_TERMINALVIEW_H
+#define EVE_TERMINALVIEW_H
+
+#include "../model/world/world.h"
+
+class TerminalView {
+private:
+    std::string formatNumberSpacing(int number, int maxLen);
+public:
+    void render(World world);
```

Comment:
change all parameters to (const Class & class)
-> e.g. render(const World & world)

Reply:
if you only say (Class class) it would copy the object, causing potential performance issues.
(Class & class) gives you the reference, i.e. the object itself which is performant, but a security issue (since you could change the underlying object)

by specifying the parameter as (const Class & class) you get a constant reference, i.e. a view of the object which you can't change

Reply:
Does this also apply to the integers of the formatNumberSpacing function?